apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/dashboard/_components/ConnectedDocumentsTable/index.tsx

```diff
@@ -0,0 +1,84 @@
+'use client'
+
+import { HEAD_COMMIT } from '@latitude-data/core/browser'
+import type { ConnectedDocumentWithMetadata } from '@latitude-data/core/repositories'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text,
+} from '@latitude-data/web-ui'
+import { formatCostInMillicents } from '$/app/_lib/formatCostInMillicents'
+import { useNavigate } from '$/hooks/useNavigate'
+import { ROUTES } from '$/services/routes'
+
+const ConnectedDocumentTableRow = ({
+  document,
+  onSelect,
+}: {
+  document: ConnectedDocumentWithMetadata
+  onSelect: () => void
+}) => {
+  return (
+    <TableRow
+      key={document.documentUuid}
+      className='cursor-pointer border-b-[0.5px] h-12 max-h-12 border-border'
+      onClick={onSelect}
+    >
+      <TableCell>
+        <Text.H4 noWrap>{document.path.split('/').pop()}</Text.H4>
+      </TableCell>
+      <TableCell>
+        <Text.H4 noWrap>{document.evaluationLogs}</Text.H4>
+      </TableCell>
+      <TableCell>
+        <Text.H4 noWrap>{document.totalTokens}</Text.H4>
+      </TableCell>
+      <TableCell>
+        <Text.H4 noWrap>
+          {formatCostInMillicents(document.costInMillicents ?? 0)}
+        </Text.H4>
+      </TableCell>
+    </TableRow>
+  )
+}
+
+export default function ConnectedDocumentsTable({
+  connectedDocumentsWithMetadata,
+}: {
+  connectedDocumentsWithMetadata: ConnectedDocumentWithMetadata[]
+}) {
+  const navigate = useNavigate()
+
+  return (
+    <Table className='table-auto'>
+      <TableHeader className='sticky top-0 z-10'>
+        <TableRow>
+          <TableHead>Name</TableHead>
+          <TableHead>Logs evaluated</TableHead>
+          <TableHead>Tokens</TableHead>
+          <TableHead>Cost</TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody className='max-h-full overflow-y-auto'>
+        {connectedDocumentsWithMetadata.map((document) => (
+          <ConnectedDocumentTableRow
+            key={document.documentUuid}
+            document={document}
+            onSelect={() =>
+              navigate.push(
+                ROUTES.projects
+                  .detail({ id: document.projectId })
+                  .commits.detail({ uuid: HEAD_COMMIT })
+                  .documents.detail({ uuid: document.documentUuid }).logs.root, // TODO: Navigate to the document evaluation details page
+              )
+            }
```

Comment:
Navigating to document logs as a mock rn. It will redirect to the document evaluation details page when it's available.

Reply:
it's available check routes in main